de4dot.code/deobfuscators/CryptoObfuscator/StringDecrypter.cs

```diff
@@ -87,6 +88,12 @@ class StringDecrypter {
 					return Encoding.UTF8.GetString(Convert.FromBase64String(s));
 				}
 				catch {
+                    string s2 = CoUtils.DecryptResourceName(module, cctor);
+                    try {
+                         return Encoding.UTF8.GetString(Convert.FromBase64String(s2));
+                    } catch {
```

Comment:
catch, just like else, should be on a new line. The empty line below is not needed.